### PR TITLE
chore(front): edit readme with figma sketch link

### PR DESCRIPTION
**Pull request by OlivPi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ You must configure the `REACT_APP_MAPBOX_ACCESS_TOKEN` environment variable in a
 nx serve frontend
 ```
 
+#### Style
+
+**Figma** sketch : `https://www.figma.com/proto/lVX7Lycox3AGixBhhbhQsQ/DatAtlas?page-id=592%3A7135&node-id=600%3A8166&viewport=727%2C1558%2C0.5&scaling=min-zoom&starting-point-node-id=600%3A8044`.
```
Comment (lutangar) [inline README.md:30]: Could you please remove the extra parameters ?
So we may land on the main page instead :
https://www.figma.com/file/lVX7Lycox3AGixBhhbhQsQ/DatAtlas

File: README.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ You must configure the `REACT_APP_MAPBOX_ACCESS_TOKEN` environment variable in a
 nx serve frontend
 ```
 
+#### Style
```
Comment (lutangar) [inline README.md:28]:
```suggestion
## Design
```

Comment (lutangar) [reply README.md:28]: Would me more appropriate as a level 2 title later in the document.
We could also mentions the company behind the design.

**Review by lutangar — CHANGES_REQUESTED** [review]

**Review by Pipazoul — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (lutangar) [reply README.md]: > Resolved via dffa02fd1592b5c6aab448313bf01d2ce68c2486

**Review by lutangar — APPROVED** [review]

**Pull request merged**